### PR TITLE
confict count

### PR DESCRIPTION
**Pull request by caspervanneck**

### CODE REVIEW

File: src/main/java/cn/nukkit/network/protocol/ClientboundMapItemDataPacket.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        this.putUnsignedVarInt(unknown);
+        this.putUnsignedVarInt(unknown2);
+        this.putVarLong(unknown3);
+        this.putByte(action);
+        this.putUnsignedVarInt(unknown4);
+        this.putVarInt(unknown5);
+        this.putByte(unknown6);
+        this.putByte(unknown7);
+        this.putBoolean(showIcons);
+        this.putUnsignedVarInt(0); // No icons yet
```
Comment (NycuRO) [inline src/main/java/cn/nukkit/network/protocol/ClientboundMapItemDataPacket.java]: putUnsignedVarInt(BinaryStream.toUInt32(new byte[]{red, green, blue, (byte) 0xff}, 0));

Comment (caspervanneck) [inline src/main/java/cn/nukkit/network/protocol/ClientboundMapItemDataPacket.java]: its not done you can edit so dont be lazy :P

Comment (NycuRO) [inline src/main/java/cn/nukkit/network/protocol/ClientboundMapItemDataPacket.java]: @caspervanneck Not now.

Comment (caspervanneck) [inline src/main/java/cn/nukkit/network/protocol/ClientboundMapItemDataPacket.java]: ???? @NycuRO

Comment (NycuRO) [inline src/main/java/cn/nukkit/network/protocol/ClientboundMapItemDataPacket.java]: I don't want to edit, i want to put code here for not dissapreal from my mind :)))))

**Pull request closed without merging**